docs/installing_deis/azure.rst

```diff
@@ -0,0 +1,121 @@
+
+Create Cluster Cloud Config
+---------------------------
+
+Before we can create a cluster, we need to create a cloud config for it. The script
+``create-azure-user-data`` does this for you. This script takes the stock cluster instance config
+in ``../coreos/user-data.example``, customizes it for Azure, and inserts a unique cluster discovery
+endpoint:
+
+.. code-block:: console
+
+    $ ./create-azure-user-data $(curl -s https://discovery.etcd.io/new)
+
+This will create a azure-user-data cloud config file. We'll use this with the script in the next
+section during cluster creation.
+
+Create CoreOS Cluster
+---------------------
+
+With the management certificate and cloud config in place, we are ready to create our cluster.
+
+* Create a container called ``vhds`` within a storage account in the same region as your cluster using the Azure portal. Note the URL of the container for the cluster creation script below.
+* Choose a cloud service name for your Deis cluster for the script below. The script will automatically create this cloud service for you.
+* Choose an Azure `region`_ to use. Supply it in quotes with the ``--location`` parameter. The default is "West US".
+
+With that, let's run the azure-coreos-cluster script which will create the CoreOS cluster. Fill in the bracketed values with the values for your deployment you created above.
+
+.. code-block:: console
+
+    $ ./azure-coreos-cluster [cloud service name]
+         --subscription [subscription id]
+         --azure-cert azure-cert.pem
+         --num-nodes 3
+         --location "[location]"
+         --vm-size Large
+         --pip
+         --deis
+         --blob-container-url https://[blob container].blob.core.windows.net/vhds/
+         --data-disk
+         --custom-data azure-user-data
+
+This script will by default provision a 3 node cluster but you can increase this with the
+``--num-nodes`` parameter. Likewise, you can increase the VM size using ``--vm-size``.
+It is not recommended that you use smaller than Large (A3) sized instances.
+
+Note that for scheduling to work properly, clusters must consist of at least 3 nodes and always
+have an odd number of members. For more information, see `optimal etcd cluster size`_.
+
+
+Configure DNS
+-------------
+
+See :ref:`configure-dns` for more information on properly setting up your DNS records with Deis.
+
+
+Install Deis Platform
+---------------------
+
+Now that you've finished provisioning a cluster, please refer to :ref:`install_deis_platform` to
+start installing the platform.
+
+.. _`management certificates tab`: https://manage.windowsazure.com/#Workspaces/AdminTasks/ListManagementCertificates
+.. _`contrib/azure`: https://github.com/deis/deis/tree/master/contrib/azure
+.. _`etcd`: https://github.com/coreos/etcd
+.. _`optimal etcd cluster size`: https://github.com/coreos/etcd/blob/master/Documentation/optimal-cluster-size.md
```

Comment:
Suggested replacement, as in #2902:  [etcd disaster recovery](https://github.com/coreos/etcd/blob/master/Documentation/admin_guide.md#disaster-recovery)